README.md

```diff
@@ -79,7 +79,7 @@ Easy correlation through ability to extract data from most popular response form
 - Several load statistics can be chosen with **pluggable tiers**.
 - Data analysis and **visualization plugins** allow great extensibility and personalization.
 - Functions can be used to provide dynamic input to a test or provide data manipulation.
-- Easy Continuous Integration via 3rd party Open Source libraries for Maven, Gradle and Jenkins.
+- Easy Continuous Integration via 3rd party Open Source libraries for Maven, Gradle, and Jenkins.
```

Comment:
Have you had a look at all the other places, where we don't follow the rule to place a comma before `and` when spelling out lists?